content/open-source/accoutrement.rst

```diff
@@ -0,0 +1,96 @@
+public: yes
```

Comment:
I don't feel strongly; just prefer not to be maintaining duplicate info. Maybe there's a way we can do a redirect? Probably not, this being a static site.